python-sdk/src/astro/extractors/facets.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict, List, Optional
+
+import attr
+from openlineage.client.facet import BaseFacet
+
+from astro.constants import FileType
+from astro.table import Column, Metadata
+
+
+@attr.s
```

Comment:
If you use the new `attr.define`, I don't think you need to put `attr.ib` everywhere below 

Check https://www.attrs.org/en/stable/api.html

Reply:
Thanks, I was not aware of this. [Done](https://github.com/astronomer/astro-sdk/pull/1025/commits/7dad9a6fc4f0bcc5205f25fe51fb0b3f629a621e)